src/screens/signup/phoneVerification/countryInput/country.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import CountryPicker, {
+  Country,
+  CountryCode
+} from 'react-native-country-picker-modal';
+import { AsYouType } from 'libphonenumber-js';
+
+import { Container, Text, PhoneInputField, CountryDailingCode } from './style';
+import ContextDisplay from '../contextDisplay/index';
+export default function InputCountry() {
+  const [country, setCountry] = useState<string>('United States');
```

Comment:
Please merge all these states into one like this:
```suggestion
interface stateTypes{
country:string;
visibility:boolean;
code:string;
dialling:string;
}
  const [state, seState] = useState<stateTypes>({
  country:"";
visibility: false;
code: "US";
dialling:"1";
  });
```